core/amber/src/main/python/core/util/print_writer/print_writer.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from typing import Iterator, List
+
+
+class SimpleBuffer:
+    def __init__(self, maximum_size=10, maximum_window=2):
+        self._max_size = maximum_size
```

Comment:
by looking at the code, I assume this is # of characters? 10 characters seems too small, I think common buffer sizes are around the scale of 1K bytes, or 512 bytes. Also please indicate the unit either in variable name or in comment.

Reply:
this is the number of lines of print messages. I just want this to be a simple implementation. Does it have to be the byte sizes? will change the variable name to make it more readable.